### PR TITLE
Convert remaining tfops in tests over to Raw.* calls, and then make additional tfops be compiler errors.

### PR DESCRIPTION
**Pull request by pschuh**
Recent PRs have all but disabled tfop. This makes it so that it is a compiler error to use it.

### CODE REVIEW

File: stdlib/public/TensorFlow/Execution.swift [hunk 1]
```diff
@@ -20,24 +20,27 @@
...
 // it takes and returns a TensorFlow value.
 @_transparent
+@available(*, deprecated)
 public func enableTPU(serverAddress: String? = nil, infeed: Bool = true) {
   _RuntimeConfig.executionMode = .tpu
   if let serverAddress = serverAddress {
     _RuntimeConfig.session = .remote(serverDef: serverAddress)
   }
-  #tfop("tfc.configureTPU", enableInfeed: infeed) as Void
+//   #tfop("tfc.configureTPU", enableInfeed: infeed) as Void
```
Comment (bgogul) [inline stdlib/public/TensorFlow/Execution.swift]: Why not remove this and the following too?

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Review by bgogul — APPROVED** [review]

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

File: stdlib/public/TensorFlow/Execution.swift (hunk 1 above)
Comment (pschuh) [reply stdlib/public/TensorFlow/Execution.swift]: Done.

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Review by saeta — APPROVED** [review]

File: lib/IRGen/IRGenSIL.cpp [hunk 2]
```diff
@@ -2000,6 +2000,7 @@ visitAutoDiffFunctionExtractInst(AutoDiffFunctionExtractInst *i) {
 // The code structure resembles
 // TFGraphFunctionLowering::visitGraphOperationInst().
 void IRGenSILFunction::visitGraphOperationInst(GraphOperationInst *i) {
+  assert(false);
```
[the remark this answers is not in the document]
Comment (pschuh) [reply lib/IRGen/IRGenSIL.cpp]: They would have to be really quick about running into it. I have a followup PR that removes this whole function.

**Comment by pschuh** [comment]
@swift-ci please test tensorflow macos

File: stdlib/public/TensorFlow/Execution.swift [hunk 3]
```diff
@@ -20,24 +20,24 @@
 // function.  We need to inline if a callee contains tensor ops, not only if
 // it takes and returns a TensorFlow value.
 @_transparent
+@available(*, deprecated)
```
Comment (rxwei) [inline stdlib/public/TensorFlow/Execution.swift]: IIRC, `deprecated` won't give you an error. `unavailable` will.

Comment (pschuh) [reply stdlib/public/TensorFlow/Execution.swift]: This can be updated to unavailable once the tests are updated.

**Pull request merged**